ai-platform-unified/tutorials/unofficial/pytorch-image-classification-custom-train-container-gpu-custom-prediction-container-torchserve/README.md

```diff
@@ -0,0 +1,243 @@
+# PyTorch Image Classification using GPU and TorchServe with Custom Container
+
+### Environment Variables
+
+```
+export PROJECT=aiplatform-dev
+export BUCKET=aiplatform-dev
+export LOCATION=us-central1
+
+echo "PROJECT: ${PROJECT}"
+echo "BUCKET: ${BUCKET}"
+echo "LOCATION: ${LOCATION}"
+```
+```
+export ML_FRAMEWORK=pytorch
+export ML_TASK=img-cls
+export TRAIN_TASK=cust-train-cont-gpu
+export PRED_TASK=cust-pred-cont-torchserve
+
+echo "ML_FRAMEWORK: ${ML_FRAMEWORK}"
+echo "ML_TASK: ${ML_TASK}"
+echo "TRAIN_TASK: ${TRAIN_TASK}"
+echo "PRED_TASK: ${PRED_TASK}"
+```
+```
+export TUTORIAL_NAME_TRAIN=${ML_FRAMEWORK}-${ML_TASK}-${TRAIN_TASK}
+export TUTORIAL_NAME_PRED=${ML_FRAMEWORK}-${ML_TASK}-${PRED_TASK}
+
+echo "TUTORIAL_NAME: ${TUTORIAL_NAME_TRAIN} for training and ${TUTORIAL_NAME_PRED} for prediction"
+```
+```
+export TUTORIAL_DIR=${PWD}
+
+echo "TUTORIAL_DIR: ${TUTORIAL_DIR}"
+```
+
+### Trainer Task Local Run
+```
+cd ${TUTORIAL_DIR}
+cd trainer && python task.py --epochs 1
+rm -rf data
+rm -rf model
+```
+
+### Custom PyTorch Container for Training
+
+```
+export HOSTNAME=gcr.io
+export IMAGE_TRAIN=${TUTORIAL_NAME_TRAIN}
+export TAG=latest
+export CONTAINER_IMAGE_URI=${HOSTNAME}/${PROJECT}/${IMAGE_TRAIN}:${TAG}
+
+echo "CONTAINER_IMAGE_URI: ${CONTAINER_IMAGE_URI}"
+```
+```
+cd ${TUTORIAL_DIR}
+cd trainer && docker build -t ${CONTAINER_IMAGE_URI} -f Dockerfile .
+```
+```
+docker run --rm ${CONTAINER_IMAGE_URI} --epochs 1
+```
+```
+docker push ${CONTAINER_IMAGE_URI}
+```
+```
+gcloud container images list --repository ${HOSTNAME}/${PROJECT}
+```
+
+### Launch a Custom Training Job
+
+##### Configs
+###### CPU
+```
+export CUST_TRAIN_DISPLAY_NAME_SDK=${TUTORIAL_NAME_TRAIN}-sdk-cpu
+
+export MACHINE_TYPE=n1-standard-4
+export ACCELERATOR_TYPE=ACCELERATOR_TYPE_UNSPECIFIED
```

Comment:
Similar question here, is it possible to skip the accelerator type when training on CPUs?